packages/@react-aria/utils/src/useObjectRef.ts

```diff
@@ -48,7 +48,7 @@ export function useObjectRef<T>(forwardedRef?: ((instance: T | null) => void) |
         forwardedRef.current = null;
       }
     };
-  }, [forwardedRef]);
+  });
```

Comment:
this will now run every render? I think that's closer to how real callback refs work, just amazed it hasn't come up before